src/cli.js

```diff
@@ -255,23 +240,23 @@ export default async function run() {
   console.log(chalk.yellow(figlet.textSync(constants.tyr.name, { horizontalLayout: 'full' })));
   winston.log('debug', preferences);
 
+  const configs = await promptConfigs();
   const prereqAnswers = await promptGlobalPrerequisites();
   const canContinue = await isUserFinishedWithPrereqs(prereqAnswers);
   if (!canContinue) {
     return;
   }
 
-  const configs = await promptConfigs();
-  initProject(configs);
+  // initialize the basic project files
+  await initProject(configs);
 
-  const githubCredentials = await promptGithubCredentials();
+  // sign the user into github
+  const credentials = await git.signIntoGithub();
```

Comment:
Can we have an option to not sign into GitHub?

Reply:
yes, but not in this issue.